src/Controller/Api/ProjectController.php

```diff
@@ -77,4 +79,26 @@ public function project(string $id, ProjectRepositoryInterface $repository, Proj
 
         return new SuccessResponse($converter->full($project));
     }
+
+    /**
+     * @param string $id
+     * @param ProjectRepositoryInterface $repository
+     * @return SuccessResponse
+     * @throws ProjectNotFoundException
+     * @throws \App\Project\Exception\InvalidProjectIdException
+     * @Route("/remove/{id}")
+     */
+    public function remove(string $id, ProjectRepositoryInterface $repository)
+    {
+        $user = $this->getUser();
+        $project = $repository->load(ProjectId::fromString($id));
+        if (!$project->canRemove($user)) {
+            throw $this->createAccessDeniedException();
+        }
+
+        $removed = $repository->remove(ProjectId::fromString($id));
+        $message = $removed ? 'Project successfully removed.' : 'Remove error.';
```

Comment:
Remove error does not say what really happened and it's probably not a SuccessResponse ;).